finalfrontier/src/config.rs

```diff
@@ -97,6 +97,21 @@ pub struct DepembedsConfig {
     pub untyped: bool,
 }
 
+/// Options for sizing vocabulary.
+#[derive(Copy, Clone, Debug, Serialize)]
+pub enum VocabCutoff {
+    /// Minimum toke count
+    ///
+    /// If MinCount is used, then no word-specific embeddings will be trained for tokens occurring
+    /// less than this count.
+    MinCount(usize),
+
+    /// Maximum target vocabulary size
+    ///
+    /// If TargetVocabSize is used, then the maximum target vocabulary size is set by this value.
```

Comment:
Replace by:

Cut off the vocabulary at the given size, retaining the *n* most frequent tokens. Tokens with the same count as the token at the cut-off point will also be discarded.